app/assets/stylesheets/govuk_publishing_components/components/govspeak/_typography.scss

```
@@ -15,6 +15,10 @@
 .gem-c-govspeak {
   @include markdown-typography;
 
+  &.gem-c-govspeak--font-size-l {
+    @include markdown-typography($base_font_size: 19, $font_size: 24, $heading_font_size: 36);
+  }
+
```

Might be better to default the class to `.gem-c-govspeak--font-size-m` and then

```suggestion
  &.gem-c-govspeak--font-size-m {
    @include markdown-typography;
  }

  &.gem-c-govspeak--font-size-l {
    @include markdown-typography($base_font_size: 19, $font_size: 24, $heading_font_size: 36);
  }
```

... so I'm not including the rules twice in the large-text case

---

Changed my mind about this - I forgot the CSS gets emmitted twice anyway with @include, so it's not like I would be reducing the file size at all.

The current approach relies on specificity, but it's a simple case so I don't think it should cause much confusion.

I'll leave this comment thread open in case there's any discussion though.

---

Yeah I think the behaviour should be that by default the govspeak component is as before, but if passed the right option, it will have the additional `gem-c-govspeak--font-size-l` class added. See my other comment about the class structure.